tests/store/filters/test_status_cases_filters.py

```diff
@@ -623,3 +624,31 @@ def test_filter_case_by_empty_internal_id(store_with_multiple_cases_and_samples:
 
     # THEN the query should return no cases
     assert cases.count() == 0
+
+
+def test_filter_cases_by_ticket_no_matching_ticket(
+    store_with_multiple_cases_and_samples: Store, non_existent_id: str
+):
+    # GIVEN a store containing cases with no matching ticket
+    cases_query: Query = store_with_multiple_cases_and_samples._get_query(table=Family)
+
+    # WHEN filtering cases by a non-existent ticket
+    filtered_cases: Query = filter_cases_by_ticket(cases=cases_query, ticket_id=non_existent_id)
+
+    # THEN the query should return no cases
+    assert filtered_cases.count() == 0
+
+
+def test_filter_cases_by_ticket_matching_ticket(
+    store_with_multiple_cases_and_samples: Store, ticket: str
```

Comment:
Add doc str
Is the fixture ticket actually ticket id?

Reply:
Yes, the fixture could have a better name. Changing it